runtime/src/accounts.rs

```diff
@@ -66,7 +67,7 @@ pub struct ErrorCounters {
 
 const ACCOUNT_DATA_FILE_SIZE: u64 = 64 * 1024 * 1024;
 const ACCOUNT_DATA_FILE: &str = "data";
-const NUM_ACCOUNT_DIRS: usize = 4;
+const NUM_ACCOUNT_DIRS: usize = 1;
```

Comment:
Is there a reason to set this to 1, having multiple paths would allow for parallel appends to happen

Reply:
Can revert this, thinking that the default case of where we are in unit tests could just use 1 for simplicity, normal case where you run the fullnode in testnet case it would pass a larger number of dirs.